### PR TITLE
vrc-get: init at 1.1.1

### PR DESCRIPTION
**Pull request by bddvlpr**
###### Description of changes

Initialized package at v1.0.1. Package was updated on 6th of June to v1.0.2. Package was updated on 19th of June to v1.1.0. Package was updated on 25th of June to v1.1.1.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bddvlpr** [comment]
My bad, in my attempt of squashing all update commits, I accidentally removed the Security framework build input. Updating it in a minute...

File: pkgs/tools/misc/vrc-get/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ fetchFromGitHub, lib, rustPlatform, pkg-config, openssl, stdenv, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vrc-get";
+  version = "v1.1.1";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/vrc-get/default.nix:5]:
```suggestion
  version = "1.1.1";
```

File: pkgs/tools/misc/vrc-get/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  src = fetchFromGitHub {
+    owner = "anatawa12";
+    repo = pname;
+    rev = version;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/vrc-get/default.nix:10]:
```suggestion
    rev = "v${version}";
```

File: pkgs/tools/misc/vrc-get/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ Security ];
+
+  # Make openssl-sys use pkg-config.
+  OPENSSL_NO_VENDOR = 1;
+
+  cargoSha256 = "0szwr8r1y7w82xfx24ffxwqa9gv8m35k30nycfn8vng54vba1rcy";
+
+  meta = with lib; {
+    description = "Open Source command line client of VRChat Package Manager, the main feature of VRChat Creator Companion (VCC), that supports Windows, Linux, and macOS.";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/vrc-get/default.nix:24]:
```suggestion
    description = "Command line client of VRChat Package Manager, the main feature of VRChat Creator Companion (VCC)";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**